liblangutil/SemVerHandler.cpp

```diff
@@ -166,6 +166,7 @@ SemVerMatchExpression SemVerMatchExpressionParser::parse()
 	catch (SemVerError const&)
 	{
 		reset();
+		return nullopt;
```

Comment:
We could also rethrow the exception, instead of returning an option.

Reply:
Fine with option